calibrate.py

```diff
@@ -87,20 +153,39 @@ def parse_args():
                         help="Invert vertical axis of the camera for the display")
     parser.add_argument("-ih", "--invertHorizontal", dest="invert_h", default=False, action="store_true",
                         help="Invert horizontal axis of the camera for the display")
-    parser.add_argument("-ep", "--maxEpiploarError", default="1.0", type=float, required=False,
-                        help="Sets the maximum epiploar allowed with rectification")
+    # parser.add_argument("-ep", "--maxEpiploarError", default="1.0", type=float, required=False,
+    #                     help="Sets the maximum epiploar allowed with rectification")
```

Comment:
Bring this back. and set the default to 0.7

Reply:
Changed.